boards/arm/nrf51dk_nrf51422/Kconfig.defconfig

```diff
@@ -1,12 +1,12 @@
-# nRF51 PCA10028 board configuration
+# nRF51 DK NRF51422 board configuration
 
 # Copyright (c) 2016 Linaro Limited.
 # SPDX-License-Identifier: Apache-2.0
 
-if BOARD_NRF51_PCA10028
+if BOARD_NRF51DK_NRF51422
 
 config BOARD
-	default "nrf51_pca10028"
+	default "nrf51dk_nrf51422"
```

Comment:
You need to also rename all `nrf51_pca10028.overlay` files.

Reply:
Yes, thanks. And not only this one, apparently.